src/protocols/secure_channel/CheckinMessage.h

```diff
@@ -45,28 +45,39 @@ class DLL_EXPORT CheckinMessage
     /**
      * @brief Generate Check-in Message payload
      *
-     * @param key       Key with which to encrypt the check-in payload
-     * @param counter   Check-in counter
-     * @param appData   Application Data to incorporate within the Check-in message. Allowed to be empty.
-     * @param output    Buffer in Which to store the generated payload. SUFFICIENT SPACE MUST BE ALLOCATED by the caller
-     *                  Required Buffer Size is : GetCheckinPayloadSize(appData.size())
+     * @note Function requires two key handles to generate the Check-In message.
+     *       Due to PSA requirements, the same key handle cannot be used for AES-CCM and HMAC-SHA-256 operations.
+     *
+     * @param[in]  aes128KeyHandle   Key handle with which to encrypt the check-in payload with the AEAD
+     * @param[in]  hmac128KeyHandle  Key handle with which to generate the nonce the check-in payload with the HMAC
+     * @param[in]  counter           Check-in counter
+     * @param[in]  appData           Application Data to incorporate within the Check-in message. Allowed to be empty.
+     * @param[out] output            Buffer in Which to store the generated payload. SUFFICIENT SPACE MUST BE ALLOCATED by the
+     * caller Required Buffer Size is : GetCheckinPayloadSize(appData.size())
      * @return CHIP_ERROR
```

Comment:
For all the @return in comments, CHIP_ERROR without description should either be accompanied with a description of errors or removed as it doesn`t add any clarification.

Reply:
done